docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -0,0 +1,122 @@
+
+In addition to [PyPI](https://pypi.org/), you can also share the packaged wheel file directly, or via a cloud storage such as AWS S3.
+
+## Package multiple modular pipelines
+
+To package multiple modular pipelines in bulk, run `kedro pipeline package --all`. This will package all pipelines specified in the `tool.kedro.pipeline.package` manifest section of the project's `pyproject.toml` file:
+
+```toml
+[tool.kedro.pipeline.package]
+first_pipeline = {alias = "aliased_pipeline", destination = "somewhere/else", env = "uat"}
+second_pipeline = {}
+```
+
+* The keys (`first_pipeline`, `second_pipeline`) are the names of the modular pipeline folders within the codebase.
+* The values are the options accepted by the `kedro pipeline package <pipeline_name>` CLI command.
+
+```eval_rst
+.. note::  Make sure `destination` is specified as a POSIX path even when working on a Windows machine.
+```
+
+## Pull a modular pipeline
+
+You can pull a modular pipeline from a wheel file by executing `kedro pipeline pull <package_name>`.
+
+* The `<package_name>` must either be a package name on PyPI or a path to the wheel file.
+* Kedro will unpack the wheel file, and install the files in following locations in your Kedro project:
+  * All the modular pipeline code in `src/<python_package>/pipelines/<pipeline_name>/`
+  * Configuration files in `conf/<env>/parameters/<pipeline_name>.yml`, where `<env>` defaults to `base`.
+  * To place parameters from a different config environment, run `kedro pipeline pull <pipeline_name> --env <env_name>`
+  * Pipeline unit tests in `src/tests/pipelines/<pipeline_name>`
+* Kedro will also parse any requirements packaged with the modular pipeline and add them to project level `requirements.in`.
+* It is advised to do `kedro build-reqs` to compile the updated list of requirements after pulling a modular pipeline.
+
+```eval_rst
+.. note::  If a modular pipeline has embedded requirements and a project ``requirements.in`` file does not already exist, it will be generated based on the project ``requirements.txt`` before appending the modular pipeline requirements.
+```
+
+You can pull a modular pipeline from different locations, including local storage, PyPI and the cloud:
+
+```eval_rst
++--------------------------------+--------------------------------------------------------------------------------------+
+| Operation                      | Command                                                                              |
++================================+======================================================================================+
+| Pulling from a local directory | ``kedro pipeline pull <project-root>/src/dist/<pipeline_name>-0.1-py3-none-any.whl`` |
++--------------------------------+--------------------------------------------------------------------------------------+
+| Pull from cloud storage        | ``kedro pipeline pull s3://my_bucket/<pipeline_name>-0.1-py3-none-any.whl``          |
```

Comment:
These should point at `.tar.gz` files, not `.whl`.

Reply:
Done